src/components/Navbar.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import logo from '../Images/planet.png';
+
+const Navbar = () => (
+  <nav>
+    <div className="app-logo">
+      <img src={logo} alt="logo" className="logo" />
+      <h1 className="app-name">Space Travelers&rsquo; Hub</h1>
+    </div>
+    <ul className="navLinks">
+      <li>
+        <NavLink to="/">
+          Rockets
+        </NavLink>
+      </li>
+      <li>
+        <NavLink to="/missions">
+          Missions
+        </NavLink>
+      </li>
+      <li>
+        <NavLink to="/profile">
+          My Profile
+        </NavLink>
+      </li>
+    </ul>
+  </nav>
```

Comment:
- [ ] @rugwizangoga and @habasm great work on your navbar, however, the color doesn't match the design color given, check the comparison below for a better understanding:

 Design  Given        |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/224472971-0a5657ff-a437-4758-9d8b-fc5a84906a9e.png) |  ![image](https://user-images.githubusercontent.com/61408860/224472987-967d1552-2daa-4fd1-844b-6f5d44098ac5.png)